std/string.d

```diff
@@ -1165,9 +1175,10 @@ ptrdiff_t lastIndexOf(Char1, Char2)(const(Char1)[] s, const(Char2)[] sub,
         in CaseSensitive cs = CaseSensitive.yes) @safe pure
     if (isSomeChar!Char1 && isSomeChar!Char2)
 {
-    import std.utf : strideBack;
+    import std.algorithm.searching : endsWith;
     import std.conv : to;
-    import std.algorithm : endsWith;
+    import std.range.primitives : front, walkLength;
+    import std.utf : strideBack;
```

Comment:
It adds unnecessary diffs, diffs that might conflict during a rebase, cheery pick, etc.